classes/Builder/Payload/Payload.php

```diff
@@ -38,7 +38,14 @@ class Payload
      */
     private $items = [];
 
-    public function setItems($array)
+    /**
+     * Setter for items
+     * Use array_replace_recursive in order to merge the new
+     * content with the previous one.
+     *
+     * @param array $array
+     */
+    public function setItems(array $array)
```

Comment:
it is not a setter anymore then, you should name it something like `addItems` or even maybe `mergeItems`

Reply:
Done ! `addAndMergeItems()` !!